### PR TITLE
Feat/helium common integration

### PR DESCRIPTION
**Pull request by Perronef5**
### 📝 Description

This PR adds a ledger-live-common integration for `helium`. 

In detail this adds:

- Adds `helium` & `helium_testnet` support to `ledger-live-desktop` & `ledger-live-mobile.`
- Add `send` transaction support.
- Add `stake` transaction support.
- Add `unstake` transaction support.
- Add `transfer stake` transaction support.
- Add `burn` transaction support.
- Add `vote` support
- Add validators table to `ledger-live-desktop`
- Add votes table to `ledger-live-desktop`

#### Testing steps

- Run `ledger-live-desktop` and add a `helium` or `helium_testnet` account.
- Need the helium testnet app on your nano s? Sideload the helium nano app using the `TESTNET=true` flag.

### ❓ Context

- **Impacted projects**: `https://github.com/LedgerHQ/app-helium` 
- **Linked resource(s)**: `https://github.com/helium/wallet-issues-tracker/issues/20` 

### ✅ Checklist

- [ ] **Test coverage** 
- [ ] **Atomic delivery** 
- [ ] **No breaking changes** 

#### Send transaction
hnt-ledger-live

#### Stake Transaction
helium-ledger-live-stake

#### Unstake Transaction
helium-ledger-live-unstake

#### Transfer Stake transaction
helium-ledger-live-transfer-stake

#### Vote Transaction (Burn)
helium-ledger-live-vote

#### Votes(Helium Improvement Proposals) Table
helium-ledger-live-hips

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

**Comment by gdutk007** [comment]
🔥🔥🔥🔥🎉🎉

**Comment by Salame1996** [comment]
When will we have this change in by? Could really use this 🙏

File: apps/cli/src/commands/getAddress.ts [hunk 1]
```diff
@@ -35,8 +35,9 @@ export default {
       derivationMode: string;
       verify: boolean;
     }>
-  ) =>
-    inferCurrency(arg).pipe(
+  ) => {
+    console.log("here!", arg);
```
Comment (Salame1996) [inline apps/cli/src/commands/getAddress.ts:39]: I think this should be removed.

Comment (Perronef5) [reply apps/cli/src/commands/getAddress.ts]: done

**Pull request closed without merging**